package-lock.json

```diff
@@ -14939,7 +14939,7 @@
     },
     "pegjs": {
       "version": "0.10.0",
-      "resolved": "https://registry.npmjs.org/pegjs/-/pegjs-0.10.0.tgz",
+      "resolved": "http://registry.npmjs.org/pegjs/-/pegjs-0.10.0.tgz",
```

Comment:
Slightly unrelated, but I wonder why these URL schemes change. I saw a PR changing `http` to `https` yesterday.

Reply:
I've no idea, good spot!